src/cache.py

```diff
@@ -14,25 +14,25 @@ class CacheService:
     def __init__(self, settings: Settings):
         self.settings = settings
         self.last_updated_at = None
-        self._session = requests.Session()
         self._cache = {}
+        self._client = httpx.AsyncClient(timeout=30)
```

Comment:
I think we'd benefit from configurable timeout.
Also, I believe the default timeout could be much shorter (e.g., 5 seconds).

Reply:
By configurable, you mean from `config.json`? Happy to add that in.

The timeout is set to 30 seconds because we were facing an abnormal number of failures with 5 seconds (which is httpx's default). Granted, we have a lot of data on Flagsmith, so our usage may not be representative of the common user. Having this configurable would support both cases.

Please confirm `config.json` is what you're thinking of, and I'll make the change.

Reply:
Should be done @khvn26, assuming I understood your ask correctly.